dts/arm/nxp/nxp_mcxw71.dtsi

```diff
@@ -245,6 +245,17 @@
 		resolution = <32>;
 		status = "disabled";
 	};
+
+	flexcan0: can@3b000 {
+		compatible = "nxp,flexcan";
+		reg = <0x3b000 0x3080>;
+		interrupts = <47 0>;
+		interrupt-names = "common";
+		clocks = <&scg SCG_K4_FIRC_CLK 0xec>;
+		clk-source = <2>;
+		sample-point = <875>;
```

Comment:
Please remove `sample-point` property. Relying on the default is better, as it adapts to the configured bitrate.

Reply:
Same with this one?